resolver/manual/manual.go

```diff
@@ -110,7 +110,7 @@ func (r *Resolver) UpdateState(s resolver.State) {
 	defer r.mu.Unlock()
 	var err error
 	if r.CC == nil {
-		panic("cannot update state as grpc.Dial with resolver has not been called")
+		panic("cannot update state as grpc.NewClient with resolver has not been called")
```

Comment:
`cannot update state as channel has not exited IDLE state`

Update throughout.

Reply:
Done